package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "build": "babel --copy-files src --out-dir dist",
-    "start": "node dist/app.js",
+    "start": "node dist/server.js",
```

Comment:
dist folder should be generated only for production.
You should include `dist` inside gitignore. It's only for production use